ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -15,11 +15,15 @@ periodics:
         set -o errexit
         failures=0
         for mapping in /etc/imagemirror/mapping_knative*; do
-          echo "Running: oc image mirror -f=$mapping --skip-multiple-scopes"
-          if ! oc image mirror -f="$mapping" --skip-multiple-scopes; then
-            echo "ERROR: Failed to mirror images from $mapping"
-            failures=$((failures+1))
-          fi
+          for count in {1..3}; do
```

Comment:
For right now, set --continue-on-error as well (assuming you have a new enough oc).  That makes sure we actually mirror everything without an error.

Reply:
this job was using quay.io/openshift/origin-cli:4.3 which didn't have it. i've updated the job to use quay.io/openshift/origin-cli:4.5 which does, and added the flag.

Reply:
this is using `quay.io/openshift/origin-cli:4.3` -- not sure why, might as well use `registry.svc.ci.openshift.org/ocp/4.5:cli`

Reply:
updated to your suggestion, @stevekuznetsov